File: Math/codeforces_116A.py

# Problem link ===>> https://codeforces.com/problemset/problem/116/A

def main():
  """Solves the Codeforces problem 116A. Tram."""
  n = int(input())
  c = 0
  m = 0
  for _ in range(n):
    a, b = map(int, input().split())
    c -= a
    c += b
    if c > m:
      m = c
  print(m)

if __name__ == '__main__':
  main()

File: Math/codeforces_116A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/116/A
// submission link ===>> https://codeforces.com/contest/116/submission/224953615

#include <iostream>
using namespace std;

int main()
{
    int n, a, b, c = 0, m = 0;
    cin >> n;
    while (n--)
    {
        cin >> a >> b;
        c -= a;
        c += b;
        if (c > m)
        {
            m = c;
        }
    }

    cout << m << endl;

    return 0;
}
